### PR TITLE
Fix minor errors

### PR DESCRIPTION
**Pull request by LemonDrew**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -577,3 +577,20 @@ testers are expected to do more *exploratory* testing.
...
           Expected: No candidate displayed on display panel. Error details shown in the result panel.
+
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+
+Team size : 5
+
+  1. To accommodate candidates from various countries, we plan to enhance the phone number input field to include area codes (e.g., +6598765432).
```
Comment (SQ77) [inline docs/DeveloperGuide.md:588]: Include this at the start:
`Add support for international phone numbers:`

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -577,3 +577,20 @@ testers are expected to do more *exploratory* testing.
...
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+
+Team size : 5
+
+  1. To accommodate candidates from various countries, we plan to enhance the phone number input field to include area codes (e.g., +6598765432). 
+     This update will allow for different international formats and enable the field to accept alphanumeric characters in the future.
+
+  2. Name Field Validation: To ensure accuracy, the name field will be enhanced to prevent the entry of numerical characters,
```
Comment (SQ77) [inline docs/DeveloperGuide.md:591]: `Improve name field validation:`

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -577,3 +577,20 @@ testers are expected to do more *exploratory* testing.
...
+
+Team size : 5
+
+  1. To accommodate candidates from various countries, we plan to enhance the phone number input field to include area codes (e.g., +6598765432). 
+     This update will allow for different international formats and enable the field to accept alphanumeric characters in the future.
+
+  2. Name Field Validation: To ensure accuracy, the name field will be enhanced to prevent the entry of numerical characters, 
+     allowing only alphabetic characters. (e.g. "John Doe 1" will not be accepted as a valid name entry)
+
+  3. Users will be able to search by job title and, making it easier to locate candidates with specific job positions
```
Comment (SQ77) [inline docs/DeveloperGuide.md:594]: Include this at the start:
`Enhance find command to take job and skills as parameters:` Users are only able to search for candidates by name currently. We plan to allow...

**Review by bigismols — APPROVED** [review]
lgtm!

**Pull request merged**